test/binaryen.js/kitchen-sink.js.txt

```diff
@@ -1418,7 +1418,7 @@ getExpressionInfo(f32.const)={"id":15,"type":3,"value":8.5}
   BinaryenConstGetValueF64(expressions[250]);
 getExpressionInfo(f64.const)={"id":15,"type":4,"value":9.5}
   {
-], expressions[160], expressions[163], expressions[166], expressions[169], expressions[172], expressions[175], expressions[178], expressions[181], expressions[184], expressions[187], expressions[190], expressions[191], expressions[192], expressions[193], expressions[195], expressions[197], expressions[198], expressions[200], expressions[202], expressions[203], expressions[204], expressions[206], expressions[212], expressions[217], expressions[224], expressions[226], expressions[228], expressions[231], expressions[233], expressions[235], expressions[237], expressions[239], expressions[240], expressions[241], expressions[242], expressions[244], expressions[245], expressions[246] };
```

Comment:
Not sure what happened there that it looked like

```c
 getExpressionInfo(f64.const)={"id":15,"type":4,"value":9.5}
   {
], expressions[160], expressions[163], expressions[166] ...
```

but it's correct now. As if any part of the output contained control characters or something.

Reply:
Hmm, [one of the Travis jobs failed](https://travis-ci.org/WebAssembly/binaryen/builds/330166865?utm_source=github_status&utm_medium=notification) (7207.5) with exactly this.

Reply:
I think this might not be random actually, the underlying issue is which emscripten version is used, as there was a bug that was just fixed - so using the fix or not leads to the difference.

But, with the limiting of long lines (the cause of the bug) on master now, that should be avoided - I hope if you merge in master here all will be well.

Reply:
At least, that's my best guess as to the cause. It is odd.